src/hexkit/providers/akafka/containers.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
```

Comment:
Just maybe rename to testcontainer.py to communicate that this is only for testing.

Reply:
Good idea. Did that now.

Reply:
Also updated the verson to 1.0.0. You need to approve again.